### PR TITLE
chore: disable countly analytics and hide unused UI

### LINKED ISSUE #2198: Remove/update countly.ipfs.tech telemetry
This is a placeholder issue: we won't have paid Countly after Feb 18th, 2024
and iiuc should remove / disable code that sends requests to countly.ipts.tech.

There is a proposal in https://github.com/ipfs-shipyard/ignite-metrics/issues/133 to update ignite-metrics to use Plausible instead and then have this repo update to use the new ignite-metrics.

Regardless of the chosen path, the Countly shutdown will happen first, independent of this work.  This means that for a time, metrics will be POSTed to countly but not stored.

cc https://github.com/ipfs/ipfs-companion/issues/1315 https://github.com/ipfs/ipfs-desktop/issues/2743

### PR DESCRIPTION
**Pull request by lidel**
This PR applies minimal changes to Close #2198 but without waiting for https://github.com/ipfs-shipyard/ignite-metrics/issues/133

- hide analytics UI components
- stop sending broken opt-out metrics to instance which no longer works

It does not remove any code, and allows restoring analytics once we have time to do https://github.com/ipfs-shipyard/ignite-metrics/issues/133.

### CODE REVIEW

**Comment by lidel** [comment]
@SgtPooki does this look ok to ship as a patch release?

File: src/bundles/analytics.js [hunk 1]
```diff
@@ -442,8 +453,9 @@ const createAnalyticsBundle = ({
       state = state || {
         lastEnabledAt: 0,
         lastDisabledAt: 0,
+        showAnalyticsComponents: DISABLE_ALL_ANALYTICS, // hide related UI  for now, see https://github.com/ipfs/ipfs-webui/issues/2198
         showAnalyticsBanner: false,
-        optedOut: false,
+        optedOut: !DISABLE_ALL_ANALYTICS, // disable analytics by default for now, see https://github.com/ipfs/ipfs-webui/issues/2198
```
Comment (SgtPooki) [inline src/bundles/analytics.js:458]:
```suggestion
        optedOut: DISABLE_ALL_ANALYTICS, // disable analytics by default for now, see https://github.com/ipfs/ipfs-webui/issues/2198
```

Comment (SgtPooki) [reply src/bundles/analytics.js:458]: don't we want to set this opted out to true for now? if not, we should probably leave as `false`

**Review by SgtPooki — APPROVED** [review]
approved, but we're currently setting `optedOut` to false

File: src/bundles/analytics.js (hunk 1 above)
Comment (lidel) [reply src/bundles/analytics.js]: Yeah my bad, I negated wrong line. Fixed in 6f36572b12ca89db88260d85d3f9c85346f067db + updated tests to confirm we hide UI and do no tracking by default.

**Pull request merged**